blob/fileblob/fileblob.go

```diff
@@ -74,15 +74,26 @@ func init() {
 }
 
 // Options sets options for constructing a *blob.Bucket backed by fileblob.
-type Options struct{}
+type Options struct {
+	URLSigner URLSigner
```

Comment:
Please add docstring for URLSigner. In particular, IIUC it is only needed if you want to use `SignedURL`, right?

Reply:
Yeah, it's only necessary if you want to use SignedURL.